src/tests/calpoker.rs

```diff
@@ -85,13 +92,13 @@ pub fn test_moves_1(allocator: &mut AllocEncoder, v1: bool) -> [GameAction; 5] {
     let bob_word = allocator
         .encode_atom(clvm_traits::Atom::Borrowed(b"0bob456789abcdef"))
         .expect("should work");
-    let alice_picks = if v1 {
+    let alice_picks = if version == CalpokerVersion::V1 {
```

Comment:
matches!(version, CalpokerVersion::V1) doesn't need an eq instance and does what you're after

Reply:
huh, thanks!